app/javascript/packages/document-capture/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@18f/identity-document-capture",
```

Comment:
do we think we'd want to call this like `@18f/identity-idp/identity-document-capture`? Or is that too much nesting?

Reply:
> do we think we'd want to call this like `@18f/identity-idp/identity-document-capture`? Or is that too much nesting?

The bigger problem is that it's not a valid package name, unfortunately. The slash can only occur once, as a separator between the scope ("18f" in this case) and the package name within that scope. Though I can see how it would be nice if it were possible, since it would make the regular expression patterns for matching include / exclude a little bit guaranteed to be exclusively to those packages within this repository.